lib/exshape/shp.ex

```diff
@@ -99,6 +100,16 @@ defmodule Exshape.Shp do
     def shape_type_to_struct(unquote(t)), do: struct!(unquote(s))
   end)
 
+  defmacrop dbg(_msg) do
```

Comment:
do you want dbg references all over this? with the comment too?

Reply:
(That's why I made it a macro; when the puts line is commented out the strings and all the formatting disappear completely from the compiled output)